components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/resources/publisher-api.yaml

```
@@ -2607,21 +2658,83 @@ paths:
-  /mcp-servers:
+  /mcp-servers/generate-from-mcp-server:
     post:
       tags:
         - MCP Servers
-      summary: Create a New MCP Server
+      summary: |
+        Create a MCP server using a third party MCP Server.
       description: |
-        This operation can be used to create a new MCP server specifying the details of the object in the payload. The 
-        new MCP server will be in `CREATED` state.
+        This operation can be used to create a MCP server using a third party MCP Server. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)